File: Python/config_reader.py
import configparser


class ConfigReader:

    def __init__(self):
        self.config = configparser.ConfigParser()
        self.config.read(
            'E:/Projects/Side Projects/LiveForexRates/Python/config.ini')

    def getwebsocketurl(self):
        return self.config['api']['websocket_url']

    def getapitoken(self):
        return self.config["api"]["token"]

    def getapikey(self):
        return self.config['api']['api_url']

    def getusersetrate(self):
        return self.config['user_settings']['user_set_rate']

    def getusersetexchange(self):
        return self.config['user_settings']['forex_symbol']

    def getusersetexchangesymbol(self):
        return self.config['user_settings']['forex_exchange_symbol']

File: Nodejs/config_reader.js
var fs = require('fs')
    , ini = require('ini')

var config = ini.parse(fs.readFileSync('./config.ini', 'utf-8'))

const getapikey = () => {
    return config.api.api_url;
}

const getusersetrate = () => {
    return config.user_settings.user_set_rate;
}

export const getapikey, getusersetrate;